src/components/Login.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from 'react';
+import Button from './Button';
+import ErrorCard from './ErrorCard';
+import Input from './Input';
+
+export default function Login() {
+  const [error, setError] = useState(null);
+  const [credentials, setCredentials] = useState({
+    email: '',
+    password: '',
+  });
+  const handleSubmit = async (e:any) => {
+    e.preventDefault();
+
+    console.log(credentials);
+  };
+
+  return (
+      <div className="h-screen flex justify-center flex-col bg-slate-50">
+        <div className="w-1/1 sm:w-1/2 lg:w-1/3 m-auto p-6 bg-white rounded-md shadow-lg">
+          <h1 className="text-center text-2xl">Login</h1>
+          <hr className="my-3" />
+          <form onSubmit={handleSubmit}>
+            <label htmlFor="email">Email</label>
+            <Input
+              name="email"
+              type="email"
+              id="email"
+              className="block my-2.5 w-full"
+              required
+              placeholder="example@example.com"
+              onChange={(e:any) => {
+                setCredentials({
+                  ...credentials,
+                  email: e.target.value,
+                });
+              }}
+            />
```

Comment:
Thinking that perhaps we want to have `label` as part of the `Input` component.

Then rename the `Input` component to `TextField` so we understand the purpose.